wait/log.go

```diff
@@ -100,7 +102,9 @@ LOOP:
 			}
 
 			logs := string(b)
-			if strings.Count(logs, ws.Log) >= ws.Occurrence {
+			if logs == "" && checkErr != nil {
```

Comment:
Should we check for error here or in L91, to avoid retrieving and reading target's logs?

Reply:
The logs just before the container stops may contain the desired entries.
Therefore, I think we should check all its logs even if the container is stopped.
To ensure that all logs read, here I confirm that the logs are empty after checking the status.